dht.go

```diff
@@ -63,7 +63,8 @@ type IpfsDHT struct {
 	strmap map[peer.ID]*messageSender
 	smlk   sync.Mutex
 
-	plk sync.Mutex
+	plk             sync.Mutex
```

Comment:
Is `plk` protecting `stripedPutLocks`? If not can we put a space between them to make it clear that they aren't related. See [mutex hat](https://dmitri.shuralyov.com/idiomatic-go#mutex-hat) pattern for explanation.

Reply:
For nits like this, it’s better for us to push the change directly and approve the PR than to force another cycle of ping pong with the contributor, along with the negative sentiment of a rejection on the basis of cosmetic nits (especially if the contributor is new and we have not formalised these preferences in a contributor guideline).

Reply:
> For nits like this, it’s better for us to push the change directly and approve the PR than to force another cycle of ping pong with the contributor

I would if I knew.

> negative sentiment of a rejection on the basis of cosmetic nits (especially if the contributor is new and we have not formalised these preferences in a contributor guideline).

This is an entirely functional suggestion and crucial to the clarity of the code. I said 'minor nitpicks' in the overall description to indicate that there was nothing that I saw that needed changes in the logic of the PR, that hadn't already been addressed. That may have been misleading, sorry.

Reply:
> I would if I knew.

We can trivially make the effort to find out where and how it's used ;-)

Reply:
Feel free to dig then ;-)

Reply:
It's still good to _teach_ contributors about conventions. One trick I use is to prefix my nits with "nit: ..." to make it clear that it isn't a correctness issue.